src/app/services/cinchy-query.service.ts

```diff
@@ -186,7 +186,7 @@ export class CinchyQueryService {
       FROM [${dataProduct}].[${table}]
       WHERE [Deleted] IS NULL
         AND [${subCol}] IS NOT NULL
-        AND trim([${subCol}]) != ''
+        AND trim('${subCol}') != ''
```

Comment:
This change is required to work with aurora environment

Reply:
@cysonya Weird. Can you please make sure QA test this still works in other scenarios.

Reply:
@cinchy-m I have made a note to test this in the "Testing" section of the Data Product model PR